src/geotools/src/test/java/org/locationtech/geogig/geotools/data/GeoGigDataStoreFactoryTest.java

```diff
@@ -63,24 +60,19 @@ public void testDataStoreFinder() throws Exception {
 
     @Test
     public void testCanProcess() {
-        final File workingDir = repoDirectory;
```

Comment:
again, no need for this, we already have the repositoryDirectory attribute as protected in the parent class.